Dockerfile

```diff
@@ -166,6 +167,12 @@ ENV PATH="/venv/bin:$PATH"
 RUN pip install poetry && poetry config virtualenvs.create false --local && \
     poetry install
 
+# # Recent version of pip is needed to install llvmlite (needed by librosa) flawlessly.
+# RUN pip$PYTHON_VER install --upgrade pip
+# # Needed because numba (for librosa) needs numpy but it seems not installed automatically with pip…  Also, llvmlite version installed by librosa won’t satisfy requirements for numba, without flexibility, so it is better to install them before…
+# RUN pip$PYTHON_VER install numpy numba
+# RUN pip$PYTHON_VER install wheel pytest pylint && python$PYTHON_VER setup.py develop
+
```

Comment:
If this is commented out, does it still need to be included?

Reply:
It seems OK now with Poetry or other induced changes…